hello-world/cli-and-ide-plugin/build.gradle

```diff
@@ -0,0 +1,13 @@
+plugins {
+    id "org.jetbrains.kotlin.jvm" version "$KOTLIN_VERSION" apply false
+    id "com.github.johnrengelman.shadow" version "$SHADOW_JAR_VERSION" apply false
+}
+
+allprojects {
+    repositories {
+        mavenCentral()
+        maven { url 'https://oss.jfrog.org/artifactory/oss-snapshot-local/' }
+        maven { url 'https://plugins.jetbrains.com/maven' }
```

Comment:
It's not necessary to specify this repository because it's the repository used by default for the Intellij Gradle Plugin.